server/src/main/java/org/elasticsearch/action/search/QueryPhaseResultConsumer.java

```diff
@@ -462,7 +463,7 @@ private long ramBytesUsedQueryResult(QuerySearchResult result) {
      * the reduce completes.
      */
     private static long estimateRamBytesUsedForReduce(long size) {
-        return Math.round(1.5d * size - size);
+        return Math.round(1.5d * size);
```

Comment:
Check the javadoc for better understanding of this method. It's being used here:

https://github.com/elastic/elasticsearch/blob/e798aa034a39d1d5a8e7bad5b14b158123311d3e/server/src/main/java/org/elasticsearch/action/search/QueryPhaseResultConsumer.java#L250-L264